module/proto/auction/v1/query.proto

```diff
@@ -22,4 +39,35 @@ service Query {
   message QueryParamsResponse {
     // params holds all the parameters of this module.
     Params params = 1 [ (gogoproto.nullable) = false ];
-  }
\ No newline at end of file
+  }
+
+message QueryAuctionPeriodsByAuctionId {
+  uint64 id = 1;
+}
+message QueryAuctionPeriodsByAuctionIdResponse {
```

Comment:
```suggestion
message QueryAuctionPeriodsByIdResponse {
```

Reply:
This is resolved at commit: [Reduce function name length](https://github.com/neitdung/gravity-bridge/commit/47b6a976af787a9da65fa3b0be93f3b8c81813da)